Some subscribers API responses do not match what Java SDK expects

It doesn't seem like the **CreateBulkSubscriberResponse** and **UnseenNotificationsCountResponse** are "deserializing" the Novu API response correctly. 

The **CreateBulkSubscriberResponse** object doesn’t error out, but the fields aren’t populated and only contain nulls despite the API response having values.

The **UnseenNotificationsCountResponse** object does cause an error. 
```
Cannot deserialize value of type `Caused by: com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot deserialize value of type `java.lang.Long` from Object value (token `JsonToken.START_OBJECT`)
 at [Source: (org.springframework.util.StreamUtils$NonClosingInputStream); line: 1, column: 9] (through reference chain: co.novu.api.subscribers.responses.UnseenNotificationsCountResponse["data"])
```

Also, **markAllSubscriberMessagesFeedAs** method errors out due to the SDK expecting Long but instead is receiving a JSON object.  
```
Caused by: com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot deserialize value of type `java.lang.Long` from Object value (token `JsonToken.START_OBJECT`)
 at [Source: (org.springframework.util.StreamUtils$NonClosingInputStream); line: 1, column: 1]
```

--- a/README.md
+++ b/README.md
@@ -437,7 +437,7 @@ public class Main {
 - `getWorkflowOverrideById(overrideId)`
 - `updateWorkflowOverride(workflowId, tenantId)`
 - `updateWorkflowOverrideById(overrideId)`
-- `deleteWorkflow(overrideId)`
+- `deleteWorkflowOverride(overrideId)`
 
 ### Notification
 

--- a/src/main/java/co/novu/api/blueprints/pojos/Blueprint.java
+++ b/src/main/java/co/novu/api/blueprints/pojos/Blueprint.java
@@ -1,4 +1,5 @@
 package co.novu.api.blueprints.pojos;
+
 import co.novu.api.common.PreferenceSettings;
 import co.novu.api.common.Step;
 import co.novu.api.common.Trigger;

--- a/src/main/java/co/novu/api/blueprints/pojos/Filters.java
+++ b/src/main/java/co/novu/api/blueprints/pojos/Filters.java
@@ -1,4 +1,6 @@
 package co.novu.api.blueprints.pojos;
+
+import com.google.gson.annotations.SerializedName;
 import lombok.Data;
 
 import java.util.List;
@@ -6,5 +8,6 @@ import java.util.List;
 @Data
 public class Filters {
    private List<String> children;
-   private String _id;
+   @SerializedName("_id")
+   private String id;
 }

--- a/src/main/java/co/novu/api/blueprints/pojos/General.java
+++ b/src/main/java/co/novu/api/blueprints/pojos/General.java
@@ -1,4 +1,5 @@
 package co.novu.api.blueprints.pojos;
+
 import lombok.Data;
 
 import java.util.List;

--- a/src/main/java/co/novu/api/blueprints/pojos/Popular.java
+++ b/src/main/java/co/novu/api/blueprints/pojos/Popular.java
@@ -1,4 +1,5 @@
 package co.novu.api.blueprints.pojos;
+
 import lombok.Data;
 
 import java.util.List;

--- a/src/main/java/co/novu/api/blueprints/pojos/Template.java
+++ b/src/main/java/co/novu/api/blueprints/pojos/Template.java
@@ -1,23 +1,32 @@
 package co.novu.api.blueprints.pojos;
+
+import com.google.gson.annotations.SerializedName;
 import lombok.Data;
 
 import java.util.List;
 
 @Data
 public class Template {
-    private String _id;
+    @SerializedName("_id")
+    private String id;
     private String type;
     private Boolean active;
     private String subject;
     private List<Content> content;
     private String contentType;
-    private String _environmentId;
-    private String _organizationId;
-    private String _creatorId;
-    private String _parentId;
-    private String _layoutId;
+    @SerializedName("_environmentId")
+    private String environmentId;
+    @SerializedName("_organizationId")
+    private String organizationId;
+    @SerializedName("_creatorId")
+    private String creatorId;
+    @SerializedName("_parentId")
+    private String parentId;
+    @SerializedName("_layoutId")
+    private String layoutId;
     private List<Variables> variables;
     private String createdAt;
     private String updatedAt;
-    private Long __v;
+    @SerializedName("__v")
+    private Long version;
 }

--- a/src/main/java/co/novu/api/blueprints/pojos/Timed.java
+++ b/src/main/java/co/novu/api/blueprints/pojos/Timed.java
@@ -1,4 +1,5 @@
 package co.novu.api.blueprints.pojos;
+
 import lombok.Data;
 
 import java.util.List;

--- a/src/main/java/co/novu/api/blueprints/pojos/Variables.java
+++ b/src/main/java/co/novu/api/blueprints/pojos/Variables.java
@@ -7,5 +7,5 @@ public class Variables {
     private String name;
     private String type;
     private Boolean required;
-    private String Id;
+    private String id;
 }

--- a/src/main/java/co/novu/api/common/NotificationGroup.java
+++ b/src/main/java/co/novu/api/common/NotificationGroup.java
@@ -18,5 +18,5 @@ public class NotificationGroup {
     private String createdAt;
     private String updatedAt;
     @SerializedName("__v")
-    private String v;
+    private String version;
 }

--- a/src/main/java/co/novu/api/common/PreferenceSettings.java
+++ b/src/main/java/co/novu/api/common/PreferenceSettings.java
@@ -1,12 +1,14 @@
 package co.novu.api.common;
 
+import com.google.gson.annotations.SerializedName;
 import lombok.Data;
 
 @Data
 public class PreferenceSettings {
     private Boolean email;
     private Boolean sms;
-    private Boolean in_app;
+    @SerializedName("in_app")
+    private Boolean inApp;
     private Boolean chat;
     private Boolean push;
 }

--- a/src/main/java/co/novu/api/common/Step.java
+++ b/src/main/java/co/novu/api/common/Step.java
@@ -1,20 +1,24 @@
 package co.novu.api.common;
 
+import com.google.gson.annotations.SerializedName;
 import lombok.Data;
 
 import java.util.List;
 
 @Data
 public class Step {
-    private String _id;
-    private String _templateId;
+    @SerializedName("_id")
+    private String id;
+    @SerializedName("_templateId")
+    private String templateId;
     private Boolean active;
     private Boolean shouldStopOnFail;
     private Object template;
     private Object replyCallback;
     private String uuid;
     private String name;
-    private String _parentId;
+    @SerializedName("_parentId")
+    private String parentId;
     private Object filters;
     private Object metadata;
 }

--- a/src/main/java/co/novu/api/environments/responses/EnvironmentResponse.java
+++ b/src/main/java/co/novu/api/environments/responses/EnvironmentResponse.java
@@ -2,19 +2,23 @@ package co.novu.api.environments.responses;
 
 import co.novu.api.environments.pojos.ApiKey;
 import co.novu.api.environments.pojos.Widget;
+import com.google.gson.annotations.SerializedName;
 import lombok.Data;
 
 import java.util.List;
 
 @Data
 public class EnvironmentResponse {
-    private String _id;
+    @SerializedName("_id")
+    private String id;
     private String name;
     private String identifier;
-    private String _organizationId;
+    @SerializedName("_organizationId")
+    private String organizationId;
     private List<ApiKey> apiKeys;
     private Widget widget;
     private String createdAt;
     private String updatedAt;
-    private String _parentId;
+    @SerializedName("_parentId")
+    private String parentId;
 }

--- a/src/main/java/co/novu/api/layouts/responses/CreateLayoutResponseData.java
+++ b/src/main/java/co/novu/api/layouts/responses/CreateLayoutResponseData.java
@@ -1,9 +1,10 @@
 package co.novu.api.layouts.responses;
 
-
+import com.google.gson.annotations.SerializedName;
 import lombok.Data;
 
 @Data
 public class CreateLayoutResponseData {
-    private String _id;
+    @SerializedName("_id")
+    private String id;
 }

--- a/src/main/java/co/novu/api/layouts/responses/LayoutResponse.java
+++ b/src/main/java/co/novu/api/layouts/responses/LayoutResponse.java
@@ -1,15 +1,20 @@
 package co.novu.api.layouts.responses;
 
+import com.google.gson.annotations.SerializedName;
 import lombok.Data;
 
 import java.util.List;
 
 @Data
 public class LayoutResponse {
-    private String _id;
-    private String _organizationId;
-    private String _environmentId;
-    private String _creatorId;
+    @SerializedName("_id")
+    private String id;
+    @SerializedName("_organizationId")
+    private String organizationId;
+    @SerializedName("_environmentId")
+    private String environmentId;
+    @SerializedName("_creatorId")
+    private String creatorId;
     private String name;
     private String description;
     private String channel;
@@ -20,5 +25,6 @@ public class LayoutResponse {
     private Boolean isDeleted;
     private String createdAt;
     private String updatedAt;
-    private String _parentId;
+    @SerializedName("_parentId")
+    private String parentId;
 }

--- a/src/main/java/co/novu/api/notifications/pojos/Job.java
+++ b/src/main/java/co/novu/api/notifications/pojos/Job.java
@@ -1,17 +1,20 @@
 package co.novu.api.notifications.pojos;
 
 import co.novu.api.common.Step;
+import com.google.gson.annotations.SerializedName;
 import lombok.Data;
 
 import java.util.List;
 
 @Data
 public class Job {
-    private String _id;
+    @SerializedName("_id")
+    private String id;
     private Object digest;
     private String status;
     private Object payload;
-    private String _notificationId;
+    @SerializedName("_notificationId")
+    private String notificationId;
     private String type;
     private String createdAt;
     private String updatedAt;

--- a/src/main/java/co/novu/api/tenants/pojos/Tenant.java
+++ b/src/main/java/co/novu/api/tenants/pojos/Tenant.java
@@ -1,11 +1,14 @@
 package co.novu.api.tenants.pojos;
 
+import com.google.gson.annotations.SerializedName;
 import lombok.Data;
 
 @Data
 public class Tenant {
-    private String _environmentId;
-    private String _id;
+    @SerializedName("_environmentId")
+    private String environmentId;
+    @SerializedName("_id")
+    private String id;
     private String createdAt;
     private Object data;
     private String identifier;

--- a/src/main/java/co/novu/api/workflowoverrides/WorkflowOverrideApi.java
+++ b/src/main/java/co/novu/api/workflowoverrides/WorkflowOverrideApi.java
@@ -1,10 +1,7 @@
 package co.novu.api.workflowoverrides;
 
-import co.novu.api.workflowoverrides.pojos.WorkflowOverride;
-
 import co.novu.api.workflowoverrides.request.CreateWorkflowOverrideRequest;
 import co.novu.api.workflowoverrides.request.UpdateWorkflowOverrideRequest;
-import co.novu.api.workflowoverrides.response.BulkWorkflowOverridesResponse;
 import co.novu.api.workflowoverrides.response.DeleteWorkflowOverrideResponse;
 import co.novu.api.workflowoverrides.response.GetWorkflowOverridesResponse;
 import co.novu.api.workflowoverrides.response.WorkflowOverrideResponse;

--- a/src/main/java/co/novu/api/workflowoverrides/WorkflowOverrideApi.java
+++ b/src/main/java/co/novu/api/workflowoverrides/WorkflowOverrideApi.java
@@ -2,8 +2,8 @@ package co.novu.api.workflowoverrides;
 
 import co.novu.api.workflowoverrides.request.CreateWorkflowOverrideRequest;
 import co.novu.api.workflowoverrides.request.UpdateWorkflowOverrideRequest;
+import co.novu.api.workflowoverrides.response.BulkWorkflowOverridesResponse;
 import co.novu.api.workflowoverrides.response.DeleteWorkflowOverrideResponse;
-import co.novu.api.workflowoverrides.response.GetWorkflowOverridesResponse;
 import co.novu.api.workflowoverrides.response.WorkflowOverrideResponse;
 import retrofit2.Call;
 import retrofit2.http.Body;
@@ -24,7 +24,7 @@ public interface WorkflowOverrideApi {
     Call<WorkflowOverrideResponse> createWorkflowOverride(@Body CreateWorkflowOverrideRequest request);
 
     @GET(ENDPOINT)
-    Call<GetWorkflowOverridesResponse> getWorkflowOverrides(@QueryMap Map<String, Object> params);
+    Call<BulkWorkflowOverridesResponse> getWorkflowOverrides(@QueryMap Map<String, Object> params);
 
     @GET(ENDPOINT + "/workflows/{workflowId}/tenants/{tenantId}")
     Call<WorkflowOverrideResponse> getWorkflowOverride(@Path("workflowId") String workflowId, @Path("tenantId") String tenantId);

--- a/src/main/java/co/novu/api/workflowoverrides/WorkflowOverrideHandler.java
+++ b/src/main/java/co/novu/api/workflowoverrides/WorkflowOverrideHandler.java
@@ -3,8 +3,8 @@ package co.novu.api.workflowoverrides;
 import co.novu.api.workflowoverrides.request.CreateWorkflowOverrideRequest;
 import co.novu.api.workflowoverrides.request.GetWorkflowOverrideRequest;
 import co.novu.api.workflowoverrides.request.UpdateWorkflowOverrideRequest;
+import co.novu.api.workflowoverrides.response.BulkWorkflowOverridesResponse;
 import co.novu.api.workflowoverrides.response.DeleteWorkflowOverrideResponse;
-import co.novu.api.workflowoverrides.response.GetWorkflowOverridesResponse;
 import co.novu.api.workflowoverrides.response.WorkflowOverrideResponse;
 import co.novu.common.rest.NovuNetworkException;
 import co.novu.common.rest.RestHandler;
@@ -30,11 +30,11 @@ public class WorkflowOverrideHandler {
         return restHandler.extractResponse(response);
     }
 
-    public GetWorkflowOverridesResponse getWorkflowOverrides(GetWorkflowOverrideRequest request) throws IOException, NovuNetworkException {
+    public BulkWorkflowOverridesResponse getWorkflowOverrides(GetWorkflowOverrideRequest request) throws IOException, NovuNetworkException {
         Map<String, Object> params = new HashMap<>();
         if (request.getPage() != null) params.put("page", request.getPage());
         if (request.getLimit() != null) params.put("limit", request.getLimit());
-        Response<GetWorkflowOverridesResponse> response = workflowOverrideApi.getWorkflowOverrides(params).execute();
+        Response<BulkWorkflowOverridesResponse> response = workflowOverrideApi.getWorkflowOverrides(params).execute();
         return restHandler.extractResponse(response);
     }
 

--- a/src/main/java/co/novu/api/workflowoverrides/WorkflowOverrideHandler.java
+++ b/src/main/java/co/novu/api/workflowoverrides/WorkflowOverrideHandler.java
@@ -1,10 +1,8 @@
 package co.novu.api.workflowoverrides;
 
-import co.novu.api.workflowoverrides.pojos.WorkflowOverride;
 import co.novu.api.workflowoverrides.request.CreateWorkflowOverrideRequest;
 import co.novu.api.workflowoverrides.request.GetWorkflowOverrideRequest;
 import co.novu.api.workflowoverrides.request.UpdateWorkflowOverrideRequest;
-import co.novu.api.workflowoverrides.response.BulkWorkflowOverridesResponse;
 import co.novu.api.workflowoverrides.response.DeleteWorkflowOverrideResponse;
 import co.novu.api.workflowoverrides.response.GetWorkflowOverridesResponse;
 import co.novu.api.workflowoverrides.response.WorkflowOverrideResponse;

--- a/src/main/java/co/novu/api/workflowoverrides/request/UpdateWorkflowOverrideRequest.java
+++ b/src/main/java/co/novu/api/workflowoverrides/request/UpdateWorkflowOverrideRequest.java
@@ -7,5 +7,5 @@ import lombok.Data;
 @Data
 public class UpdateWorkflowOverrideRequest {
     private Boolean active;
-    private PreferenceSettings data;
+    private PreferenceSettings preferenceSettings;
 }

--- a/src/main/java/co/novu/api/workflowoverrides/response/GetWorkflowOverridesResponse.java
+++ b/src/main/java/co/novu/api/workflowoverrides/response/GetWorkflowOverridesResponse.java
@@ -1,8 +1,0 @@
-package co.novu.api.workflowoverrides.response;
-
-import lombok.Data;
-
-@Data
-public class GetWorkflowOverridesResponse {
-    private BulkWorkflowOverridesResponse data;
-}

--- a/src/main/java/co/novu/common/base/Novu.java
+++ b/src/main/java/co/novu/common/base/Novu.java
@@ -108,13 +108,11 @@ import co.novu.api.workflowgroups.responses.DeleteWorkflowGroup;
 import co.novu.api.workflowgroups.responses.GetWorkflowGroupsResponse;
 import co.novu.api.workflowgroups.responses.WorkflowGroupResponse;
 import co.novu.api.workflowoverrides.WorkflowOverrideHandler;
-import co.novu.api.workflowoverrides.pojos.WorkflowOverride;
 import co.novu.api.workflowoverrides.request.CreateWorkflowOverrideRequest;
 import co.novu.api.workflowoverrides.request.GetWorkflowOverrideRequest;
 import co.novu.api.workflowoverrides.request.UpdateWorkflowOverrideRequest;
 import co.novu.api.workflowoverrides.response.BulkWorkflowOverridesResponse;
 import co.novu.api.workflowoverrides.response.DeleteWorkflowOverrideResponse;
-import co.novu.api.workflowoverrides.response.GetWorkflowOverridesResponse;
 import co.novu.api.workflowoverrides.response.WorkflowOverrideResponse;
 import co.novu.api.workflows.WorkflowHandler;
 import co.novu.api.workflows.requests.UpdateWorkflowRequest;
@@ -996,11 +994,11 @@ public class Novu {
         }
     }
 
-    public GetWorkflowOverridesResponse getWorkflowOverrides(GetWorkflowOverrideRequest request) throws IOException, NovuNetworkException {
+    public BulkWorkflowOverridesResponse getWorkflowOverrides(GetWorkflowOverrideRequest request) throws IOException, NovuNetworkException {
         try {
             return workflowOverrideHandler.getWorkflowOverrides(request);
         } catch (Exception e) {
-            log.error("Error fetching Workflow-Override", e);
+            log.error("Error fetching bulk Workflow-Override", e);
             throw e;
         }
     }
@@ -1018,7 +1016,7 @@ public class Novu {
         try {
             return workflowOverrideHandler.getWorkflowOverrideById(overrideId);
         } catch (Exception e) {
-            log.error("Error fetching Workflow-Override", e);
+            log.error("Error fetching Workflow-Override by id", e);
             throw e;
         }
     }
@@ -1027,7 +1025,7 @@ public class Novu {
         try {
             return workflowOverrideHandler.updateWorkflowOverrideById(overrideId, request);
         } catch (Exception e) {
-            log.error("Error updating Workflow-Override", e);
+            log.error("Error updating Workflow-Override by id", e);
             throw e;
         }
     }

--- a/src/test/java/co/novu/api/environments/EnvironmentHandlerTest.java
+++ b/src/test/java/co/novu/api/environments/EnvironmentHandlerTest.java
@@ -40,17 +40,17 @@ public class EnvironmentHandlerTest extends TestCase {
 
         SingleEnvironmentResponse singleEnvironmentResponse = new SingleEnvironmentResponse();
         EnvironmentResponse data = new EnvironmentResponse();
-        data.set_id("id");
+        data.setId("id");
         data.setName("name");
         data.setIdentifier("identifier");
-        data.set_organizationId("organizationId");
+        data.setOrganizationId("organizationId");
         data.setApiKeys(Collections.singletonList(new ApiKey()));
         Widget widget = new Widget();
         widget.setNotificationCenterEncryption(false);
         data.setWidget(widget);
         data.setCreatedAt("createdAt");
         data.setUpdatedAt("updatedAt");
-        data.set_parentId("parentId");
+        data.setParentId("parentId");
         singleEnvironmentResponse.setData(data);
         
         Gson gson = new Gson();
@@ -72,17 +72,17 @@ public class EnvironmentHandlerTest extends TestCase {
 
         SingleEnvironmentResponse singleEnvironmentResponse = new SingleEnvironmentResponse();
         EnvironmentResponse data = new EnvironmentResponse();
-        data.set_id("id");
+        data.setId("id");
         data.setName("name");
         data.setIdentifier("identifier");
-        data.set_organizationId("organizationId");
+        data.setOrganizationId("organizationId");
         data.setApiKeys(Collections.singletonList(new ApiKey()));
         Widget widget = new Widget();
         widget.setNotificationCenterEncryption(false);
         data.setWidget(widget);
         data.setCreatedAt("createdAt");
         data.setUpdatedAt("updatedAt");
-        data.set_parentId("parentId");
+        data.setParentId("parentId");
         singleEnvironmentResponse.setData(data);
         
         Gson gson = new Gson();
@@ -124,17 +124,17 @@ public class EnvironmentHandlerTest extends TestCase {
 
         SingleEnvironmentResponse singleEnvironmentResponse = new SingleEnvironmentResponse();
         EnvironmentResponse data = new EnvironmentResponse();
-        data.set_id("id");
+        data.setId("id");
         data.setName("name");
         data.setIdentifier("identifier");
-        data.set_organizationId("organizationId");
+        data.setOrganizationId("organizationId");
         data.setApiKeys(Collections.singletonList(new ApiKey()));
         Widget widget = new Widget();
         widget.setNotificationCenterEncryption(false);
         data.setWidget(widget);
         data.setCreatedAt("createdAt");
         data.setUpdatedAt("updatedAt");
-        data.set_parentId("parentId");
+        data.setParentId("parentId");
         singleEnvironmentResponse.setData(data);
         
         Gson gson = new Gson();

--- a/src/test/java/co/novu/api/layouts/LayoutHandlerTest.java
+++ b/src/test/java/co/novu/api/layouts/LayoutHandlerTest.java
@@ -48,7 +48,7 @@ public class LayoutHandlerTest extends TestCase {
 
         CreateLayoutResponse createLayoutResponse = new CreateLayoutResponse();
         CreateLayoutResponseData createLayoutResponseData = new CreateLayoutResponseData();
-        createLayoutResponseData.set_id("id");
+        createLayoutResponseData.setId("id");
         createLayoutResponse.setData(createLayoutResponseData);
 
         Gson gson = new Gson();

--- a/src/test/java/co/novu/api/workflowoverride/WorkflowOverrideHandlerTest.java
+++ b/src/test/java/co/novu/api/workflowoverride/WorkflowOverrideHandlerTest.java
@@ -136,7 +136,7 @@ public class WorkflowOverrideHandlerTest extends TestCase {
         preferenceSettings.setSms(true);
         preferenceSettings.setPush(true);
         preferenceSettings.setEmail(true);
-        preferenceSettings.setIn_app(true);
+        preferenceSettings.setInApp(true);
         return preferenceSettings;
     }
 
@@ -180,7 +180,7 @@ public class WorkflowOverrideHandlerTest extends TestCase {
     private UpdateWorkflowOverrideRequest getUpdateWorkflowOverrideRequest() {
         UpdateWorkflowOverrideRequest updateWorkflowOverrideRequest = new UpdateWorkflowOverrideRequest();
         updateWorkflowOverrideRequest.setActive(true);
-        updateWorkflowOverrideRequest.setData(getPreferenceSettings());
+        updateWorkflowOverrideRequest.setPreferenceSettings(getPreferenceSettings());
         return updateWorkflowOverrideRequest;
     }
 

--- a/src/test/java/co/novu/api/workflowoverride/WorkflowOverrideHandlerTest.java
+++ b/src/test/java/co/novu/api/workflowoverride/WorkflowOverrideHandlerTest.java
@@ -8,7 +8,6 @@ import co.novu.api.workflowoverrides.request.GetWorkflowOverrideRequest;
 import co.novu.api.workflowoverrides.request.UpdateWorkflowOverrideRequest;
 import co.novu.api.workflowoverrides.response.BulkWorkflowOverridesResponse;
 import co.novu.api.workflowoverrides.response.DeleteWorkflowOverrideResponse;
-import co.novu.api.workflowoverrides.response.GetWorkflowOverridesResponse;
 import co.novu.api.workflowoverrides.response.WorkflowOverrideResponse;
 import co.novu.common.base.NovuConfig;
 import co.novu.common.rest.NovuNetworkException;
@@ -52,13 +51,13 @@ public class WorkflowOverrideHandlerTest extends TestCase {
     }
 
     public void test_getWorkflowOverrides() throws IOException, NovuNetworkException, InterruptedException {
-        GetWorkflowOverridesResponse workflowOverridesResponse = getWorkflowOverridesResponse();
+        BulkWorkflowOverridesResponse workflowOverridesResponse = getWorkflowOverridesResponse();
         Gson gson = new Gson();
         mockWebServer.enqueue(new MockResponse().setResponseCode(200).setBody(gson.toJson(workflowOverridesResponse)));
         GetWorkflowOverrideRequest getWorkflowOverrideRequest = new GetWorkflowOverrideRequest();
         getWorkflowOverrideRequest.setPage(1);
         getWorkflowOverrideRequest.setLimit(10);
-        GetWorkflowOverridesResponse response = workflowOverrideHandler.getWorkflowOverrides(getWorkflowOverrideRequest);
+        BulkWorkflowOverridesResponse response = workflowOverrideHandler.getWorkflowOverrides(getWorkflowOverrideRequest);
         assertNotNull(response);
         RecordedRequest request = mockWebServer.takeRequest();
         assertEquals("/workflow-overrides?limit=10&page=1", request.getPath());
@@ -166,15 +165,13 @@ public class WorkflowOverrideHandlerTest extends TestCase {
         return createWorkflowOverrideRequest;
     }
 
-    private GetWorkflowOverridesResponse getWorkflowOverridesResponse() {
+    private BulkWorkflowOverridesResponse getWorkflowOverridesResponse() {
         BulkWorkflowOverridesResponse bulkWorkflowOverridesResponse = new BulkWorkflowOverridesResponse();
         bulkWorkflowOverridesResponse.setData(List.of(getWorkflowOverride(), getWorkflowOverride()));
         bulkWorkflowOverridesResponse.setHasMore(true);
         bulkWorkflowOverridesResponse.setPageSize(10L);
         bulkWorkflowOverridesResponse.setPage(1L);
-        GetWorkflowOverridesResponse getWorkflowOverridesResponse = new GetWorkflowOverridesResponse();
-        getWorkflowOverridesResponse.setData(bulkWorkflowOverridesResponse);
-        return getWorkflowOverridesResponse;
+        return bulkWorkflowOverridesResponse;
     }
 
     private UpdateWorkflowOverrideRequest getUpdateWorkflowOverrideRequest() {

--- a/src/test/java/co/novu/api/workflows/WorkflowHandlerTest.java
+++ b/src/test/java/co/novu/api/workflows/WorkflowHandlerTest.java
@@ -67,7 +67,7 @@ public class WorkflowHandlerTest extends TestCase {
         PreferenceSettings preferenceSettings = new PreferenceSettings();
         preferenceSettings.setEmail(true);
         preferenceSettings.setSms(true);
-        preferenceSettings.setIn_app(true);
+        preferenceSettings.setInApp(true);
         preferenceSettings.setPush(true);
         preferenceSettings.setChat(true);
         workflowRequest.setPreferenceSettings(preferenceSettings);
@@ -94,7 +94,7 @@ public class WorkflowHandlerTest extends TestCase {
         PreferenceSettings preferenceSettings1 = new PreferenceSettings();
         preferenceSettings1.setEmail(true);
         preferenceSettings1.setSms(true);
-        preferenceSettings1.setIn_app(true);
+        preferenceSettings1.setInApp(true);
         preferenceSettings1.setPush(true);
         preferenceSettings1.setChat(true);
         data.setPreferenceSettings(preferenceSettings1);
@@ -140,7 +140,7 @@ public class WorkflowHandlerTest extends TestCase {
         PreferenceSettings preferenceSettings = new PreferenceSettings();
         preferenceSettings.setEmail(true);
         preferenceSettings.setSms(true);
-        preferenceSettings.setIn_app(true);
+        preferenceSettings.setInApp(true);
         preferenceSettings.setPush(true);
         preferenceSettings.setChat(true);
         workflowRequest.setPreferenceSettings(preferenceSettings);
@@ -165,7 +165,7 @@ public class WorkflowHandlerTest extends TestCase {
         PreferenceSettings preferenceSettings1 = new PreferenceSettings();
         preferenceSettings1.setEmail(true);
         preferenceSettings1.setSms(true);
-        preferenceSettings1.setIn_app(true);
+        preferenceSettings1.setInApp(true);
         preferenceSettings1.setPush(true);
         preferenceSettings1.setChat(true);
         data.setPreferenceSettings(preferenceSettings1);
@@ -229,7 +229,7 @@ public class WorkflowHandlerTest extends TestCase {
         PreferenceSettings preferenceSettings1 = new PreferenceSettings();
         preferenceSettings1.setEmail(true);
         preferenceSettings1.setSms(true);
-        preferenceSettings1.setIn_app(true);
+        preferenceSettings1.setInApp(true);
         preferenceSettings1.setPush(true);
         preferenceSettings1.setChat(true);
         data.setPreferenceSettings(preferenceSettings1);
@@ -277,7 +277,7 @@ public class WorkflowHandlerTest extends TestCase {
         PreferenceSettings preferenceSettings1 = new PreferenceSettings();
         preferenceSettings1.setEmail(true);
         preferenceSettings1.setSms(true);
-        preferenceSettings1.setIn_app(true);
+        preferenceSettings1.setInApp(true);
         preferenceSettings1.setPush(true);
         preferenceSettings1.setChat(true);
         data.setPreferenceSettings(preferenceSettings1);

